src/__tests__/useA11yBlockLink.test.js

```diff
@@ -0,0 +1,144 @@
+import React, {useRef} from 'react';
+import {render, configure, fireEvent} from '@testing-library/react';
+import useA11yBlockLink from '../useA11yBlockLink';
+
+configure({testIdAttribute: 'data-test-id'});
+
+const Demo = ({testId, handleMainClick, handleThirdClick}) => {
+  const mainClickRef = useRef();
+  const secondRef = useRef();
+  const thirdRef = useRef();
+
+  const {handleClick: handleA11yClick} = useA11yBlockLink({
+    mainClick: mainClickRef,
+    secondRef,
+    thirdRef,
```

Comment:
Will these be used for something in the future? Seems like `mainClick` is the only one used 🤔 .

Thoughts on calling `mainClick` `target` instead? IMO that might convey a bit more about that being the element that is wrapping the other interactive content